pkg/edgexsdk/edgexsdk.go

```diff
@@ -48,6 +50,8 @@ type AppFunctionsSDK struct {
 	configProfile  string
 	configDir      string
 	useRegistry    bool
+	errChannel     chan error
```

Comment:
Don't use the type in the name. How about "httpErrors"
Had this discussion about not using types in names with Trevor. Here is a good blog about it.
https://dave.cheney.net/practical-go/presentations/qcon-china.html#_dont_name_your_variables_for_their_types